Automattic/wp-calypso
# i18n: Support @wordpress/i18n translation functions

#### Changes proposed in this Pull Request

See pbAok1-dn-p2

* Setup locale in relevant places for `@wordpress/i18n` to be used in Gutenboarding and Calypso in general.

See non-HEAD sha for demonstration of functionality - http://hash-060e4cf06ea0ab8903cdd4844132b2b9d7546778.calypso.live/gutenboarding

#### Testing instructions

* The masterbar should show the correct language in Calypso: http://hash-060e4cf06ea0ab8903cdd4844132b2b9d7546778.calypso.live/ (navigate between reader and my-sites to get the components to rerender with loaded language)
* http://hash-060e4cf06ea0ab8903cdd4844132b2b9d7546778.calypso.live/log-in/es should show Spanish UI.
* http://es.calypso.localhost:3000/gutenboarding should show in Spanish (checkout 060e4cf06ea0ab8903cdd4844132b2b9d7546778)

Closes …
Closes #38044
